app/app.go

```diff
@@ -459,6 +459,7 @@ func (app *App) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.Respo
 	res := app.manager.EndBlock(ctx, req)
 	currentVersion := app.AppVersion()
 	// For v1 only we upgrade using an agreed upon height known ahead of time
+	app.BaseApp.Logger().Debug("current version", "version", currentVersion)
```

Comment:
Don't you think this is a little too spammy (I know it's debug logs) to have the version printed for each block?

Reply:
Yea fair enough, it's mostly for my debugging but prob not useful for node operators. I can remove this line and add it back when I debug version issues.